dbt/include/global_project/macros/materializations/external_table/external_table.sql

```diff
@@ -0,0 +1,58 @@
+{% materialization external_table, default %}
+
+  {%- set identifier = model['alias'] -%}
+  {%- set full_refresh_mode = (should_full_refresh()) -%}
+
+  {%- set existing_relation = load_cached_relation(this) -%}
```

Comment:
I need help here understanding why `existing_relation.type` returns `None` when the existing relation is an External Table